utils/docker/orakwlum-frontend/start_frontend.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+# Set Server Name
+if [ "$#" -ne 1 ]; then
+    def_serv_name="localhost"
+    echo -n "Enter a server name to patch [$def_serv_name]: "
+    read server_name
+    if [ "$server_name" = "" ]; then
+        server_name=$def_serv_name
+    fi
+else
+    server_name="$1"
+fi
+name="frontend"
+container="okw/"$name
+ports="-p 8080:80 -p 443:443"
+socket_path="/tmp/docker_sockets"
+if [ -d $socket_path ]; then
+    mkdir -p $socket_path
+fi
+# Ensure permissions on socket
+chmod -R 777 $socket_path
+sockets="-v $socket_path:/tmp/"
+log_path="/var/log/nginx/frontend_docker"
+if [ -d $log_path ]; then
+    mkdir -p $log_path
+fi
+logs="-v $log_path:/var/log/orakwlum"
```

Comment:
Based on the container's nginx path:
https://github.com/gisce/oraKWlum-frontend/blob/master/conf/production.conf#L10

So now, the updated log path (host's path) will be on /var/log/orakwlum/frontend